src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -65,10 +65,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('templating_twig.xml');
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
-
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
-            $loader->load('security_debug.xml');
-        }
+        $loader->load('security_debug.xml');
```

Comment:
Yes but now the check is done in the collector, so don't need to load the debug add when not in debug mode.